engine/env/spark/src/main/java/org/datacleaner/spark/Main.java

```diff
@@ -39,20 +56,53 @@ public static void main(String[] args) {
         final String confXmlPath = args[0];
         final String analysisJobXmlPath = args[1];
 
-        final String propertiesPath; 
+        final String propertiesPath;
         if (args.length > 2) {
             propertiesPath = args[2];
         } else {
             propertiesPath = null;
         }
 
-        final SparkJobContext sparkJobContext = new SparkJobContext(sparkContext, confXmlPath, analysisJobXmlPath, propertiesPath);
-
+        final SparkJobContext sparkJobContext = new SparkJobContext(sparkContext, confXmlPath, analysisJobXmlPath,
+                propertiesPath);
         final SparkAnalysisRunner sparkAnalysisRunner = new SparkAnalysisRunner(sparkContext, sparkJobContext);
         try {
-            sparkAnalysisRunner.run();
+            final AnalysisResultFuture result = sparkAnalysisRunner.run();
+            if (result.isDone()) {
+                final Map<ComponentJob, AnalyzerResult> resultMap = result.getResultMap();
+                final SimpleAnalysisResult simpleAnalysisResult = new SimpleAnalysisResult(resultMap,
+                        result.getCreationDate());
+                final String resultJobFilePath = getResultJobFilePath(sparkContext, sparkJobContext);
+                if (resultJobFilePath != null) {
+                    final HdfsResource hdfsResource = new HdfsResource(resultJobFilePath);
```

Comment:
Oh, I guess HDFSResource would still handle it if it weren't, given that it's always parsable by the FS. Ignore :)